docs/template_plugin/backend/backend.cpp

```diff
@@ -0,0 +1,47 @@
+// Copyright (C) 2018-2021 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#include "backend.hpp"
+
+#include <sstream>
+
+#include "int_backend.hpp"
+#include "ngraph/file_util.hpp"
+#include "ngraph/util.hpp"
+#include "openvino/util/file_util.hpp"
+
+using namespace std;
+using namespace ngraph;
+
+std::mutex runtime::Backend::m_mtx;
+std::string runtime::Backend::s_backend_shared_library_search_directory;
```

Comment:
is `s_backend_shared_library_search_directory` used?

Reply:
Removed